docs/multichain.md

```diff
@@ -2,20 +2,192 @@
+its Merkle proof**, at the given height. The light client may store all previous
+headers that it has accepted so far to verify the inclusion from past blocks.
 
 ## Trustless Message Delivery
 
+A light client takes blockchain headers and their proofs as the only input to
+update its state. In other words, the light client can stay in sync with the
+blockchain without accessing the network if there are constant header updates
+from whoever submits valid ones.
+
+Thanks to this simple but powerful property, it is possible to embed a light
+client as a smart contract. This enables **trustless message delivery between
+two different blockchains**, which is the key to multichain interoperability.
+
+Suppose there are two different blockchains called A and B. If the A chain has a
+light client of the B Chain deployed on A chain, whatever finalized on the B
+chain can be instantly verified on the A chain. Other contracts deployed on the
+A chain would interact with the B chain since the light client provides data
+from the B chain after verification. This communication can trivially work both
+ways if the B chain has a light client of the A chain as well.
+
+Cosmos IBC (Inter-Blockchain Communication) is an example of a protocol based on
+this idea. IBC defines multiple layers over the core communication primitive
+(two different chains embedding light clients of each other) to establish
+high-level communication such as asset transfer. Various chains built with
+Cosmos SDK (a.k.a. zones) are all standalone layer-1 networks, but they form an
+organic multichain structure using IBC.
+
+## Settlement Chain
+
+Simperby is a standalone blockchain that has its own consensus mechanism and
+distributed file system. However, there must be working applications that
+interact with the Simperby chain so that the organization can finalize something
+meaningful. Of course, the most important applications would be other existing
+chains that the organization deploys their applications on. From Simperby's
+view, they are called **settlement chains**. Any blockchain that has a smart
+contract platform can be a settlement chain once the organization pays some gas
+and deploys their light client on it.
+
+Simperby's scalability over settlement chain integration is quite impressive;
+the source code for the light client contract will be almost the same for every
+chain except for a few virtual machine-specific behaviors and the host
+interface. This enables **scalable development of light clients over various
+settlement chains**, which eventually provides powerful multichain
+interoperability for the DAO that uses Simperby.
+
+Also, considering that a Simperby chain has no contracts or business logic but
+is only capable of recording explicitly approved data, there is nothing to
+execute automatically from the Simperby side. That is, the multichain
+communication system **doesn't have to be two-way**. Things happening
```

Comment:
If I understand correctly, I think ```bidirectional``` is better than ```two-way```

Reply:
fixed.